CommunityToolkit.HighPerformance/Extensions/ReadOnlySpanExtensions.cs

```diff
@@ -208,7 +208,7 @@ public static ReadOnlySpan2D<T> AsSpan2D<T>(this ReadOnlySpan<T> span, int offse
     /// <typeparam name="T">The type if items in the input <see cref="ReadOnlySpan{T}"/>.</typeparam>
     /// <param name="span">The input <see cref="ReadOnlySpan{T}"/> to calculate the index for.</param>
     /// <param name="value">The reference to the target item to get the index for.</param>
-    /// <returns>The index of <paramref name="value"/> within <paramref name="span"/>.</returns>
+    /// <returns>The index of <paramref name="value"/> within <paramref name="span"/>, or <c>-1</c>.</returns>
     /// <exception cref="ArgumentOutOfRangeException">Thrown if <paramref name="value"/> does not belong to <paramref name="span"/>.</exception>
```

Comment:
shouldn't this be removed too?

Reply:
Whoops! Fixed in https://github.com/CommunityToolkit/dotnet/pull/190/commits/76f339fa8805af02092496bff65492c16d4a2342, thanks! 😄